pallets/dapp-staking/src/benchmarking/utils.rs

```diff
@@ -180,14 +180,17 @@ pub(super) fn init_tier_settings<T: Config>() {
             TierThreshold::DynamicPercentage {
                 percentage: Perbill::from_parts(11_112_000), // 1.1112%
                 minimum_required_percentage: Perbill::from_parts(8_889_000), // 0.8889%
+                maximum_possible_percentage: None,
```

Comment:
For the sake of simplicity & readability, I suggest to change this to Perbill and use 100% as the default value, as issue suggests.